internal/logger/logger.go

```diff
@@ -0,0 +1,65 @@
+package logger
+
+import (
+	"fmt"
+	"log"
+)
+
+var isDebugMode bool
+
+// EnableDebugMode method enables verbose logging.
+func EnableDebugMode() {
+	isDebugMode = true
+
+	Debug("Enable verbose logging")
+}
+
+// Debug method logs message with "debug" level.
+func Debug(a ...interface{}) {
+	if !isDebugMode {
+		return
+	}
+	logMessage("DEBUG", a...)
+}
+
+// Debugf method logs message with "debug" level and formats it.
+func Debugf(format string, a ...interface{}) {
+	if !isDebugMode {
+		return
+	}
+	logMessagef("DEBUG", format, a...)
+}
+
+// Info method logs message with "info" level.
+func Info(a ...interface{}) {
+	logMessage("INFO", a...)
+}
+
+// Infof method logs message with "info" level and formats it.
+func Infof(format string, a ...interface{}) {
+	logMessagef("INFO", format, a...)
+}
+
+// Warn method logs message with "warn" level.
+func Warn(a ...interface{}) {
+	logMessage("WARN", a...)
+}
+
+// Warnf method logs message with "warn" level and formats it.
+func Warnf(format string, a ...interface{}) {
+	logMessagef("WARN", format, a...)
+}
+
+func logMessage(level string, a ...interface{}) {
+	var all []interface{}
+	all = append(all, fmt.Sprintf("%5s ", level))
+	all = append(all, a...)
+	log.Print(all...)
```

Comment:
Should we include a timestamp in the log message? Not sure how valuable they will be as most sub-commands tend to run for relatively short periods of time. Also, there's no concurrency yet so establishing order of operations via log messages is quite straightforward even without timestamps. Just thought I'd ask and get your thoughts on the idea.

Reply:
Hm.. If you don't ming I would rather keep them there. Timestamps may help us finding some bottlenecks in processing large integrations, interacting with Kibana instance or similar. It's better to have them instead of regret :)

Reply:
Ha, I totally misunderstood that timestamps were already included in the message! I though we would have to add them manually here that's why I even brought up the question. If they are already included then ++ to just keeping them!